jobbergateapi2/tests/apps/users/test_schemas.py

```diff
@@ -59,8 +59,8 @@ def test_create_user(user_data):
     user = User(**user_data)
 
     assert user.email == user_data["email"]
-    assert user.is_admin is False
-    assert user.username == user_data["username"]
+    assert user.is_superuser is False
+    assert user.full_name == user_data["full_name"]
     assert user.Config.orm_mode is True
```

Comment:
We do not check for orm_mode when creating users previously. Not sure if that will be important.

Reply:
Each test must be contained in testing just one thing, as a rule of thumb. So in this test we are testing only the schema for the User, so testing it here that the orm_mode is True, is enough, that is already tested here, so we can be sure that always when creating a User schema it has the orm_mode as True